packages/legend-studio/src/components/editor/edit-panel/uml-editor/AssociationEditor.tsx

```diff
@@ -80,9 +84,25 @@ const AssociationPropertyBasicEditor = observer(
   }) => {
     const { association, property, selectProperty, isReadOnly } = props;
     const editorStore = useEditorStore();
+    const checkDuplicateValue = (val: string, id: string): void => {
+      if (
+        association.properties[0].name === val ||
+        association.properties[1].name === val
+      ) {
+        guaranteeNonNullable(
+          document.getElementById(id),
+        ).style.backgroundColor = 'red';
```

Comment:
:( no, let's not do this, let's use a boolean and set the style on the element. Did you check how we do that in `ServiceExecutionEditor` for runtime and mapping?

Reply:
If you think the styling of the input box is looking weird in light theme. I have a screenshot of how it looks like without that background color. Do you want me to change the styling to this?
![pic2](https://user-images.githubusercontent.com/87973126/142867814-ad2e8ad1-d424-4d18-bf65-d43590c9bf1b.PNG)